packages/analytics-js-plugins/src/deviceModeDestinations/index.ts

```diff
@@ -71,11 +71,7 @@ const DeviceModeDestinations = (): ExtensionPlugin => ({
       const configSupportedDestinations =
         state.nativeDestinations.configuredDestinations.value.filter((configDest: Destination) => {
           // Filter enabled custom destinations that don't have an integration added to them
-          if (
-            configDest.enabled &&
-            configDest.isCustomIntegration &&
-            isUndefined(configDest.integration)
-          ) {
+          if (configDest.isCustomIntegration && isUndefined(configDest.integration)) {
```

Comment:
For disabled custom integrations, only config and status can be overridden.
So, without knowing whether the destination status is overridden (to enabled) we cannot log a warning to the console.

Source configuration override is an optional feature.

Reply:
Yes, makes sense.